py3status/parse_config.py

```diff
@@ -441,6 +442,12 @@ def parse(self, dictionary=None, end_token=None):
                     dictionary[name] = value
                 # assignment of value
                 elif t_value == '=':
+                    # if we have a colon in the name of a setting then it
+                    # indicates that it has been encoded.
+                    if ':' in name:
+                        (name, scheme) = name.split(':')
+                        if scheme == 'base64':
```

Comment:
if this ought to be extendable, I'd rather have this done in a dedicated method please